### DISCUSSION TITLE
Formatação de Código no Fórum

### DISCUSSION THREAD

**Discussion by DouglasValerio** [Flutterando/forum#15] [Dicas]
O github suporta markdown, então, podemos facilitar a vida de quem lê as perguntas ou respostas usando:

<img width="410" alt="image" src="https://user-images.githubusercontent.com/16308684/189720716-64feb953-58d1-4f26-9614-b51ac26ac365.png">

Que vai resultar num bloco formatadinho de código, como o abaixo:

```dart
Widget build(BuildContext context) {
  return Scaffold(
    appBar: AppBar(title: const Text('Empty List Test')),
    body: itemCount > 0
      ? ListView.builder(
          itemCount: itemCount,
          itemBuilder: (BuildContext context, int index) {
            return ListTile(
              title: Text('Item ${index + 1}'),
            );
          },
        )
      : const Center(child: Text('No items')),
  );
}
```

Assim, fica mais fácil de ler e entender as informações.

**Comment by jacobaraujo7** [comment]
Movido para categoria "Dicas" que acabou de ser criada :)
